pkg/apis/build/v1alpha1/source.go

```diff
@@ -8,11 +8,24 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+// Container describes the source code bundle container to pull
+type Container struct {
```

Comment:
I do wonder if `Container` as the name of a struct type is overloaded, given its use in core k8s

how opposed would everyone be to a more precise name like `ImageBundleContainer` or `BundleImageContainer` ?

Reply:
Or `BundleContainer` as I see a `BundleConatinerTemplate` field below